ynr/apps/people/models.py

```diff
@@ -33,6 +33,13 @@ def person_image_path(instance, filename):
     return "images/people/{0}/{1}".format(instance.person.id, filename)
 
 
+@unique
+class EditLimitationStatuses(Enum):
+    NO_STATUS = ""
```

Comment:
Is there a way to keep the value in the enum as `""` but give it a nicer label in the dropdown in `/admin` ? If its not immediately obvious, we can ignore  this.

Reply:
Actually, I think I can just remove that? There isn't a point in a constant that is an empty string, and the default on the DB field is an empty string. Without it, the admin interface looks like:

![Screenshot 2019-09-10 at 18 23 15](https://user-images.githubusercontent.com/242329/64635728-33727500-d3f8-11e9-965e-a9dbdd9f476d.png)